packages/dart_frog_auth/lib/src/dart_frog_auth.dart

```diff
@@ -195,6 +199,7 @@ Middleware cookieAuthentication<T extends Object>({
     Map<String, String> cookies,
   ) authenticator,
   Applies applies = _defaultApplies,
+  Handler? unauthenticatedResponse,
```

Comment:
as @alestiago said [here](https://github.com/dart-frog-dev/dart_frog/pull/1632/files#r1900701836), I prefer renaming to `onUnauthorized`
```suggestion
  Handler? onUnauthorized,
```
since we're returning an unauthorized status by default, but I'm also good with onUnauthenticated